### DISCUSSION TITLE
[Bug] request.query 没有setter了？ 是个bug.

### DISCUSSION THREAD

**Discussion by pandaApe** [midwayjs/midway#3841] [Q&A] [answered]
### Describe the problem（描述问题）

`ctx.request.query = {}`
报错 ` Cannot set property query of #<Object> which has only a getter`

### Midway Versions（Midway 版本）

v3.16.0

**Comment by czy88840616** [comment]
koa ?

**Reply by pandaApe** [reply]
Yes, correct.

**Reply by czy88840616** [reply]
https://github.com/midwayjs/midway/pull/3842

**Reply by pandaApe** [reply]
Thanks!

**Reply by czy88840616** [reply, marked as answer]
@midwayjs/koa@3.16.1

**Reply by pandaApe** [reply]
https://github.com/midwayjs/midway/pull/3842#discussion_r1594913025
